#! /bin/sh /usr/share/dpatch/dpatch-run                                                                                                                                            
## 03-fix-TAO-logWalker by Marek Brudka <mbrudka@aster.pl>                                                                                                                         
##                                                                                                                                                                                 
## All lines beginning with `## DP:' are a description of the patch.                                                                                                               
## DP: change base project setting in MPC
--- ACE_wrappers.orig/TAO/utils/logWalker/logWalker.mpc	2009-06-12 18:14:23.000000000 +0200
+++ ACE_wrappers/TAO/utils/logWalker/logWalker.mpc	2010-03-21 20:29:04.000000000 +0100
@@ -1,7 +1,7 @@
 // -*- mpc -*-
 // $Id: logWalker.mpc 85624 2009-06-12 16:14:23Z mesnier_p $
 
-project : aceexe, avoids_ace_for_tao {
+project : taoexe, avoids_ace_for_tao {
   exename = tao_logWalker
   install = $(ACE_ROOT)/bin
 }
